### PR TITLE
Update manpage Examples section

### PR DESCRIPTION
**Pull request by mwichmann**
**Examples**: mainly syntax: adding/adjusting markup, reformatting code examples, etc.
This is a doc-only change.

Signed-off-by: Mats Wichmann <mats@linux.com>

## Contributor Checklist:

* [ ] I have created a new test or updated the unit tests to cover the new/changed functionality.
* [ ] I have updated `CHANGES.txt` (and read the `README.rst`)
* [X] I have updated the appropriate documentation

### CODE REVIEW

File: doc/man/scons.xml [hunk 1]
```diff
@@ -6940,37 +6945,40 @@ you can leave off the target file suffix,
...
-<para>Note also that the above initialization
-overwrites the default Builder objects,
-so the Environment created above
-can not be used call Builders like
-<methodname>env.Program</methodname>,
-<methodname>env.Object</methodname>,
-<methodname>env.StaticLibrary</methodname> etc.</para>
+<para>Note that the above initialization
+replaces the default dictionary of Builders,
+so this &consenv; can not be used call Builders like
```
Comment (bdbaddog) [inline doc/man/scons.xml:6962]: Should this be "can not be used to call Builders..."

Comment (mwichmann) [reply doc/man/scons.xml:6962]: I seem to be blind, isn't that what it says?

Comment (bdbaddog) [reply doc/man/scons.xml:6962]: Nope.. it's missing "used **to** call Builders..."

Comment (bdbaddog) [reply doc/man/scons.xml:6962]: I'll merge and fix.

Comment (bdbaddog) [reply doc/man/scons.xml:6962]: Ahh. misread... xml blinded.. ;)

Comment (mwichmann) [reply doc/man/scons.xml:6962]: no, i missed the "to"

**Pull request merged**